wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneResolver.swift

```diff
@@ -107,7 +107,19 @@ public final class OneOnOneResolver: OneOnOneResolverInterface {
                 else {
                     return
                 }
+
+                let selfUser = ZMUser.selfUser(in: context)
+
+                if !conversation.isForcedReadOnly {
```

Comment:
I understood from the PR title that we **should** display the system message if the conversation is `isForcedReadOnly`, no?

Reply:
I will add a comment on top of the if statement to make things more clear. 👍